src/main/java/org/apache/sling/mom/package-info.java

```diff
@@ -20,5 +20,7 @@
 /**
  */
 
-@org.osgi.annotation.versioning.Version("1.0.0")
+@Version("1.0.1")
```

Comment:
Why did you upgrade the version to 1.0.1?

Reply:
suggested by `bnd-baseline-maven-plugin`
```[INFO] The baseline version was found to be 1.0.0
[ERROR] Baseline mismatch for package org.apache.sling.mom, MICRO change. Current is 1.0.0, repo is 1.0.0, suggest 1.0.1 or -
MICRO                PACKAGE    org.apache.sling.mom
  REMOVED            ANNOTATED  aQute.bnd.annotation.Version
    REMOVED          PROPERTY   value='1.0.0'
  ADDED              ANNOTATED  org.osgi.annotation.versioning.Version
    ADDED            PROPERTY   value='1.0.0'```

Reply:
Got it. A bit annoying, but I guess a workaround is too much work compared to just bumping the patch component of the version, which should be mostly harmless.